src/js/backend.ts

```diff
@@ -115,6 +118,38 @@ export class ReactNativeBackend extends BaseBackend<BrowserOptions> {
     }
 
     this._options.onReady?.({ didCallNativeInit });
+
+    if (didCallNativeInit) {
+      const {
+        nativeStartTime,
+        nativeStartedTime,
+      } = await NATIVE.getNativeStartupTimestamps();
+
+      const nativeStartTimeSeconds = nativeStartTime / 1000;
+      const nativeStartedTimeSeconds = nativeStartedTime / 1000;
+
+      const t = getCurrentHub().startTransaction({
+        name: "App Start",
+        description: "App Start",
+        op: "startup",
+        startTimestamp: nativeStartTimeSeconds,
+        sampled: true,
+      });
+
+      t.startChild({
+        description: "Native App Start",
+        op: "native-startup",
+        startTimestamp: nativeStartTimeSeconds,
+      }).finish(nativeStartedTimeSeconds);
```

Comment:
is there a finish method that takes an interval in JS? I guess its not a documented API and something we'd like to have too